public/base-locales/de.json

```diff
@@ -432,18 +432,18 @@
   "profile_error": "Muss eindeutig und gültig sein",
   "team_0": "Neutral",
   "battle_changes": "im Kampf",
-  "battle_changes_poracle": "battle_changes",
+  "battle_changes_poracle": "wird angegriffen",
   "auto": "Auto",
-  "webhooks_subtitle": "Verwalten Deine Benachrichtigungen für diese Bots direkt von der Karte aus",
+  "webhooks_subtitle": "Verwalten deine Benachrichtigungen für diese Bots direkt von der Karte aus",
```

Comment:
Sorry one more `Verwalte deine`

Reply:
was changed to small letter a last commit :-)